lib/facter/apt_reboot_required.rb

```diff
@@ -0,0 +1,7 @@
+# apt_reboot_required.rb
+Facter.add(:apt_reboot_required) do
+  confine :osfamily => 'Debian'
+  setcode do
+    File.file?('/var/run/reboot-required') ? ( "true" ) : ( "false" )
```

Comment:
Please return `true` and `false` the booleans, not `"true"` and `"false"` the strings. This matters for Facter 2 and the stringify facts behaviour.